docs/dev-notes/DEVELOPMENT.md

```diff
@@ -21,7 +21,7 @@ To get started with the project, you might need several tools to compile on your
    - Load rbenv in your shell by running `rbenv init` and restarting the shell
    - Make sure you have the required ruby version set locally. You can do this by running `rbenv local 2.7.6`
  - Make sure you have [Xcode Command Line Tools installed](https://stackoverflow.com/a/45566089)
- - Make sure you have JDK installed on your machine. Recommended JDK versions is 11, if you encounter issues with Java/Kotlin files compilation, please upgrade/downgrade JDK version to the recommended one.
+ - Make sure you have [recommended JDK version](https://reactnative.dev/docs/environment-setup#installing-dependencies:~:text=on%20your%20system%2C-,we%20recommend%20JDK,-11.%20You%20may) installed on your machine. If you encounter issues with Java/Kotlin files compilation, please upgrade/downgrade JDK version to the recommended one.
```

Comment:
Thanks or adding a URL. I don't think it's working for me because when I visit the link, I don't see anything about JDK on the whole webpage? 

I will assume it's something wrong with my machine. If you're confident that the URL is correct, I approve.

Reply:
Ahh. I figured it out 😅 It was failing for me as well in incognito. And it was because, there are tabs and similar element ids and the browser was unable to identify the exact section I wanted to scroll to. I've updated link to mention tabs. It also includes the highlight text which may only work for chrome, for other browsers, it should scroll to the nearest heading with id. Let me know if this still doesn't work for you 😬